app/build.gradle

```diff
@@ -38,7 +38,7 @@ android {
     }
     defaultConfig {
         applicationId "org.greenstand.android.TreeTracker"
-        minSdkVersion 15
+        minSdkVersion 21
```

Comment:
This app is for 3rd world countries. Do we really want to ignore users with old phones?